a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -115,133 +95,327 @@ test.serial('remove IST', async t => {
   );
 });
 
-test.serial('close vault', async t => {
-  const activeVaults = await agopsVaults(USER1ADDR);
-  const vaultPath = activeVaults[activeVaults.length - 1];
-  const vaultID = vaultPath.split('.').pop();
+test.serial('add collateral', async t => {
+  const { vaultID, collateral: collateralBefore } =
+    await getLastVaultFromAddress(USER1ADDR);
+  t.log('vault collateral before:', collateralBefore);
+
+  await adjustVault(USER1ADDR, vaultID, { giveCollateral: 1.0 });
 
-  let vaultData = await getContractInfo(vaultPath, { agoric, prefix: '' });
-  const vaultCollateral = vaultData.locked.value;
-  t.log('vault collateral:', vaultCollateral);
+  const { collateral: collateralAfter } =
+    await getLastVaultFromAddress(USER1ADDR);
+  t.log('vault collateral after:', collateralAfter);
+
+  t.is(
+    collateralBefore,
+    collateralAfter - 1_000_000n,
```

Comment:
I know the math is the same, but this would be easier for reviewers to parse.
```suggestion
    collateralAfter,
    collateralBefore + 1_000_000n,
```

Reply:
Change request addressed